src/components/DeleteModal/DeleteModal.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import Button from '@mui/material/Button';
+import ClearIcon from '@mui/icons-material/Clear';
+import CheckIcon from '@mui/icons-material/Check';
+
+const DeleteModal = ({showDeleteModal,setShowDeleteModal,client,handleDelete}) => (
+    <Dialog
+    open={showDeleteModal}
+    aria-labelledby="dialog-title"
+    aria-describedby="dialog-description"
+    onClose={() => setShowDeleteModal(false)}
+  >
+   
+   <DialogTitle id="dialog-tile">Delete Client?</DialogTitle>
+   
+   <DialogContent>
+      <DialogContentText id="dialog-description">
+      {`You're about to delete ${client.person} from your client list, do you wish to continue?`}
+      </DialogContentText>
+    </DialogContent>
+   
+    <DialogActions>
+      <Button
+        variant="outlined"
+        color="error"
+        endIcon={<ClearIcon />}
+        onClick={() => setShowDeleteModal(false)}
+      >
+        NO
+      </Button>
+
+        <Button
+          variant="outlined"
+          color="success"
+          endIcon={<CheckIcon />}
+          sx={{ marginLeft: '1rem' }}
+          onClick={handleDelete}
```

Comment:
This `handleDelete` which is really `handleDeleteClient`....

**1)** I personally would prefer the naming stays detailed (part of self-documenting code, "what are we deleting? a client").    When working with a larger team, especially remotely that doesn't meetup as often, we want to keep the code specific, so every dev who encounters the codebase easily understands what exactly is happening.

**2)** This onClick function doesn't change the `showDeleteModal` state, so when a user clicked this button to approve deleting a client the modal will not go away - which is unexpected behavior and makes the user have to click a different button afterwords or click outside of the modal.

Reply:
I will Change or name the function and file to be more specific